### PR TITLE
add image upload and file upload

### PR DESCRIPTION
**Pull request by HanqingZhou**
Add image and file upload:
Screenshot 2023-10-17 at 2 13 13 PM

### CODE REVIEW

File: src/components/FileUpload/FileUpload.js
Comment (troychaplin) [inline src/components/FileUpload/FileUpload.js:1]: I removed extra divs we no longer need. Please review changes.

File: src/components/ImageUpload/ImageUpload.js
Comment (troychaplin) [inline src/components/ImageUpload/ImageUpload.js:1]: I removed extra divs we no longer need. Please review changes.

**Review by troychaplin — APPROVED** [review]
Review the changes and merge to develop. Nice work!

File: src/components/FileUpload/FileUpload.js
Comment (HanqingZhou) [reply src/components/FileUpload/FileUpload.js:1]: Looks great, thank you, forgot to remove the unused one

File: src/components/ImageUpload/ImageUpload.js
Comment (HanqingZhou) [reply src/components/ImageUpload/ImageUpload.js:1]: Looks great, thank you, now the image and file upload are in the wrapper, no extra div needed

**Comment by HanqingZhou** [comment]
Going to merge to develop, thanks a lot

**Pull request merged**